packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -1,6 +1,7 @@
 import { StatementSyntax, ValueReference } from 'glimmer-runtime';
 import { AttributeBindingReference, RootReference, applyClassNameBinding } from '../utils/references';
 import EmptyObject from 'ember-metal/empty_object';
+import Dict from 'ember-metal/dictionary';
```

Comment:
We probably don't need it to be an actual dict since we know all the fields (like meta). We can just make an actual class for it